src/Parsing/Impl/Definition/IScopeStatement.cs

```diff
@@ -0,0 +1,7 @@
+using Microsoft.Python.Parsing.Ast;
+
+namespace Microsoft.Python.Parsing.Definition {
```

Comment:
namespace

Reply:
If it is a folder, should be `Definitions` in the folder name, but no `Definitions` namespace.

Reply:
OK